custom_components/moonraker/button.py

```diff
@@ -93,6 +93,14 @@ class MoonrakerButtonDescription(ButtonEntityDescription):
         ),
         icon="mdi:refresh",
     ),
+    MoonrakerButtonDescription(
+        key="reset_totals",
+        name="Reset Totals",
+        press_fn=lambda button: button.coordinator.async_send_data(
```

Comment:
This is a very precise feature, and I think can lead to more error than use. Can you put it as disabled by default. (Folks will have to manually add it) 

```suggestion
        entity_registry_enabled_default=False,
        press_fn=lambda button: button.coordinator.async_send_data(
```